services/data/methods.go

```diff
@@ -540,7 +540,7 @@ func (d *dataClient) CloseTrain(train *types.Train, override bool) error {
 	train.ScheduleOverride = override
 	_, err := d.Client.Update(train, "Closed", "ScheduleOverride")
 	if err == nil {
-		datadog.Info("Closed train %v", train)
+		datadog.Info("Closed train (ID, HeadSHA, TailSHA) %v", train.ID, train.HeadSHA, train.TailSHA)
```

Comment:
Does this work? You only have one %v. Will that expand to all three somehow?

Reply:
Ditto to all the other ones like this in the review.

Reply:
I don't think it'll work. https://play.golang.org/p/Ylg9gunfZO2

Reply:
Fixed